bitnami/jupyterhub/templates/hub/secret.yaml

```diff
@@ -20,4 +20,7 @@ data:
   proxy-token: {{ default (randAlphaNum 32) .Values.proxy.secretToken | b64enc | quote }}
   hub.config.JupyterHub.cookie_secret: {{ include "jupyterhub.hub.config.JupyterHub.cookie_secret" . | b64enc | quote }}
   hub.config.CryptKeeper.keys: {{ include "jupyterhub.hub.config.CryptKeeper.keys" . | b64enc | quote }}
+  {{- range $key, $service := .Values.hub.services }}
```

Comment:
`hub.services` is not defined in the `values.yaml` file. It should be defined and it would be great if you could add a configuration example.